code/datums/mind.dm

```diff
@@ -62,6 +62,8 @@
 
 	var/force_escaped = FALSE  // Set by Into The Sunset command of the shuttle manipulator
 
+	var/hijack_speed = 0			//whether or not we can hackerman the shuttle and hijack it. 0 means you can't, 1 is normal speed hack, 2 is double, 0.5 is half, etc.
```

Comment:
good lord just use a component. hell, use an element. hell, you're already adding it to the damn antag datum, just USE THE ANTAG DATUM, there's not much good reason to have this duplicated

Reply:
i'm saying this cause i want to reduce the mind/mob bloat at some point and this adds to it and DAMN